content/query.md

```diff
@@ -0,0 +1,3 @@
+# Querying
+
+Query Content

```

Comment:
content altına markdown örneklerini koymuştuk, nuxt'tan yazılan page'ler için demo klasörü var, oraya alalım bunu örn. demo/provide-and-inject.md